python/tvm/relax/transform/lazy_transform_params.py

```diff
@@ -118,9 +119,15 @@ class LazyTransformParamsMutator(PyExprMutator):
         The module to be transformed
     """
 
-    def __init__(self, mod: IRModule = None) -> None:
+    def __init__(
+        self, fget_item, fset_item, get_item_param, set_item_param, mod: IRModule = None
```

Comment:
From the usage, the `get_item_param` and `set_item_param` are lists of elements, so they should be plural `get_item_params` and `set_item_params`.  (If they are required overall.  See other comment asking if they are.)

This would also be an advantage of breaking the functionality apart into two separate mutators: A `LazyInput` mutator could have an `extra_params` argument, and a `LazyOutput` mutator could have an `extra_params` argument, and both would be clear from the context.  With both mutations implemented in the same class, if we wanted the name to indicate that these are additional arguments, we'd need to make it be `extra_get_item_params` and `extra_set_item_params`, which is a bit long for ease of use.

Reply:
I cannot fully separate these mutators because they need to share `input_tuple_param` and func.params[0] will change after one mutator.

Reply:
so what I can do is create a class that does all the analysis, calls into the two mutators, and create function. In this case, we have to use `extra_get_item_params` and `extra_set_item_params`

Reply:
Hmm, looks like the analysis would need to be split apart as well.  In general, I think having multiple smaller changes is better than a single mutator, but this seems like a refactor than would make sense to ask in a feature addition PR.  The current revision looks good, and any additional changes can be in a follow-up PR if desired.